packages/kbn-openapi-generator/src/openapi_linter.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable no-console */
+
+import { join, resolve } from 'path';
+import globby from 'globby';
+import execa from 'execa';
+import chalk from 'chalk';
+
+export interface LinterConfig {
+  rootDir: string;
+  sourceGlob: string;
+}
+
+export const lint = async (config: LinterConfig) => {
+  const { rootDir, sourceGlob } = config;
+
+  const sourceFilesGlob = resolve(rootDir, sourceGlob);
+  const schemaPaths = await globby([sourceFilesGlob]);
+
+  console.log(chalk.bold(`Linting API route schemas`));
+
+  try {
+    await execa('npx', ['@redocly/cli', 'lint', '--config=config.yaml', ...schemaPaths], {
```

Comment:
Instead of using `npx`, call the installed CLI directly:

```suggestion
    await execa('../../../node_modules/.bin/redocly', ['lint', '--config=config.yaml', ...schemaPaths], {
```